widgetastic_manageiq/__init__.py

```diff
@@ -1089,6 +1091,119 @@ def workaround_save_issue(self):
             "{}._handlers.change.map(function(handler) {{ handler() }});".format(self.item_name))
 
 
+class SSUIVerticalNavigation(VerticalNavigation):
+    """The Patternfly Vertical navigation."""
+    CURRENTLY_SELECTED = './/li[contains(@class, "active")]/a/span'
+
+    @property
+    def currently_selected(self):
+        for el in self.browser.elements(self.CURRENTLY_SELECTED, parent=self):
+            return self.browser.text(el)
```

Comment:
REQUIRED: `parent=self` should be unnecessary and this loop will return the first value or none. If that is the intent, I would then rather prefer using `[0]` and catching indexerror in here.

Which brings me to the fact that I saw view is_displayed matchers that matched currently_selected to a list, whereas this should not return a list afaiu, so those will always return false.

Reply:
This method is not required. Removed